frontend/src/components/AuditLog.vue

```diff
@@ -0,0 +1,254 @@
+                        </i18n-t>
+                      </div>
+                      <div class="flex flex-1 justify-between sm:justify-end">
+                        <button type="button" class="relative inline-flex items-center rounded-md bg-white px-3 py-2 text-sm font-medium text-gray-900 ring-1 ring-inset ring-gray-300 hover:bg-gray-50 focus-visible:outline-offset-0 disabled:opacity-50 disabled:hover:bg-white disabled:cursor-not-allowed" :disabled="currentPage == 1" @click="showPreviousPage()">
+                          {{ t('common.previous') }}
+                        </button>
+                        <button type="button" class="relative ml-3 inline-flex items-center rounded-md bg-white px-3 py-2 text-sm font-medium text-gray-900 ring-1 ring-inset ring-gray-300 hover:bg-gray-50 focus-visible:outline-offset-0 disabled:opacity-50 disabled:hover:bg-white disabled:cursor-not-allowed" :disabled="!hasNextPage" @click="showNextPage()">
+                          {{ t('common.next') }}
+                        </button>
+                      </div>
+                    </nav>
+                  </td>
+                </tr>
+              </tfoot>
+            </table>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { Popover, PopoverButton, PopoverGroup, PopoverPanel } from '@headlessui/vue';
+import { ChevronDownIcon } from '@heroicons/vue/20/solid';
+import { computed, onMounted, ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+import backend, { AuditEventDto } from '../common/backend';
+import FetchError from './FetchError.vue';
+
+const { t, d } = useI18n({ useScope: 'global' });
+
+const auditEvents = ref<AuditEventDto[]>();
+const onFetchError = ref<Error | null>();
+
+const startDate = ref(beginOfDate(new Date()));
+const startDateFilter = ref(startDate.value.toISOString().split('T')[0]);
+const endDate = ref(endOfDate(new Date()));
+const endDateFilter = ref(endDate.value.toISOString().split('T')[0]);
+
+const filterIsReset = computed(() => {
+  return startDateFilter.value == startDate.value.toISOString().split('T')[0]
+    && endDateFilter.value == endDate.value.toISOString().split('T')[0];
+});
+const startDateFilterIsValid = computed(() => {
+  if (!/^\d{4}-\d{2}-\d{2}$/.test(startDateFilter.value)) {
+    return false;
+  }
+  const startDate = new Date(startDateFilter.value);
+  return !isNaN(startDate.getTime());
+});
+const endDateFilterIsValid = computed(() => {
+  if (!/^\d{4}-\d{2}-\d{2}$/.test(endDateFilter.value)) {
+    return false;
+  }
+  const endDate = new Date(endDateFilter.value);
+  if (isNaN(endDate.getTime())) {
+    return false;
+  }
+  if (startDateFilterIsValid.value) {
+    const startDate = new Date(startDateFilter.value);
+    return startDate <= endDate;
+  }
+  return true;
+});
+const filterIsValid = computed(() => {
+  return startDateFilterIsValid.value && endDateFilterIsValid.value;
+});
+
+const currentPage = ref(1);
```

Comment:
Better start with the logical numbering (zero based indexing) and just adjust the ui, otherwise you to always substract or add 1 in the code, decreasing readability.

Reply:
Fixed in d7fff0aa8bd4108d09dbfacb14315161dd90108f